test/models/contact_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ContactTest < ActiveSupport::TestCase
```

Comment:
If you're using RSpec you should remove the minitest automatically generated files.